content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -53,3 +64,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+Before running the payslips, you may adjust the accounts used in the salary rule by navigating to
+:menuselection:`Payroll --> Configurations --> Rules`.
+
+.. image:: hong_kong/hk-salary-rules.png
+   :alt: Hong Kong Salary Rules
+
+Odoo can create pay runs in **two** ways: via **individual** or via **batch** payslips.
+
+.. _hong_kong/batch_payslips:
+
+Batch payslips
+~~~~~~~~~~~~~~
+
+Go to :menuselection:`Payroll --> Payslips --> Batches`
+This method of payslip generation is used for recurring payments as you can manage the payslip for
+multiple employees at once.
+
+#. Click on :guilabel:`New`.
+#. Provide a Batch Name (e.g, `2024 – Jan`) and Period (e.g. 01/01/2024 - 01/31/2024)
+#. Click on :guilabel:`Generate Payslips`.
+#. You can choose which **Salary Structure** you wish to use for this batch. The department filter
+   allows you to designate the batch for a specific group of employees.
+#. Click on :guilabel:`Generate`.
+#. Now you can see the :guilabel:`Payslips` smart button created automatically.
+
+.. image:: hong_kong/hk-batch-payslips.png
+   :alt: Hong Kong Batch Payslips
+
+Once you are satisfied with the Payslips, click :guilabel:`Create Draft entry` to generate a draft
+journal entry found in the :guilabel:`Other Info` tab of each payslips.
+
+Individual payslips
+~~~~~~~~~~~~~~~~~~~
+
+Go to :menuselection:`Payroll --> Payslips --> All Payslips`
+This method of payslip generation is commonly used to handle one-off payments (e.g. Payment in Lieu
+of Notice, Long Service Payment, Severance Payment).
+
+#. Click on :guilabel:`New`.
+#. Select an :guilabel:`Employee`; their :guilabel:`Contract` will be filled out automatically
+#. Add a pay :guilabel:`Period`
+#. Select a salary :guilabel:`Structure` (e.g. Employees Monthly Pay)
+#. The **Worked Days** tab will automatically compute the worked days/hours and time-off leaves
+   that are applicable.
+#. You may also choose to add additional payslip items (e.g. Commissions, Deductions) under the
+   **Other Inputs** section.
+#. Click on :guilabel:`Compute Sheet` button to generate the payslip lines. This button will update
+   the :guilabel:`Salary Computation` tab.
+
+.. image:: hong_kong/hk-individual-payslip.png
+   :alt: Hong Kong Individual Payslip
+
+.. note::
+   If you have amended the work entry for an employee, you will need to go to the **gear icon (⚙)**
+   and click on **Recompute Whole Sheet** to refresh the payslip's **Worked Day & Inputs** section.
+
+The :guilabel:`Salary Computation` tab will show the detailed breakdown of the computation based on
+the salary rules configured for each structure type.
+
+.. image:: hong_kong/hk-salary-computation.png
+   :alt: Hong Kong Salary computation
+
+#. **Rent Allowance**: Amount derived from the employee's active rental record.
+#. **Basic salary**: Amount from which tax-deductible lines can be subtracted minus rental
+   allowance.
+#. **713 Gross**: Amount after considering allowances/deductions.
+#. **MPF Gross**
```

Comment:
this is not defined- there should be a short explanation for everything the list.